common/djangoapps/nodebb/tasks.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from celery.exceptions import MaxRetriesExceededError
+from celery.task import task  # pylint: disable=no-name-in-module, import-error
+
+from common.lib.nodebb_client.client import NodeBBClient
+from django.conf import settings
+
+log = logging.getLogger('edx.celery.task')
+
+
+@task(bind=True, default_retry_delay=settings.NODEBB_TASK_DELAY, max_retries=settings.NODEBB_TASK_MAX_TRIES)
+def create_category_on_nodebb_task(self, course_name):
```

Comment:
I think its better to prefix a task method with `task_` instead of suffixing it with `_task`.